docs/installation/reference/import.rst

```diff
@@ -0,0 +1,177 @@
+request containing the csv data in its request body.
+
+Whenever the csv contains invalid and or missing headers the import process will
+not be started. Whenever headers are missing the error response will contain those
+headers.
+
+One column which is required for each row in the csv file is the path to the
+file which will be imported and will be assosiated to the `EnkelvoudigInformatieObject`.
+This should be a relative path from the path configured in `IMPORT_DOCUMENTEN_BASE_DIR`.
+
+Before starting an import, the directory configured through `IMPORT_DOCUMENTEN_BASE_DIR`,
+is accessed to check if this directory exists (and if it actually is a directory).
+Whenever this is not the case the import also will not start.
+
+Just like in the step to create the `Import` instance, no other `pending` or `active`
+import instances can be active before starting the specified `Import`.
+
+Whenever this request passes validation (with the above mentioned checks) the
+actual import process is started through a background task.
+
+**Retrieving the status**
+
+Whenever an `Import` instance has the status `pending`, `active`, `finished` or
+`error` and the import process is started, the status of that `Import` can be
+retrieved. This endpoint can be called through a `GET` request. The data of the
+response contains information for example about the total amount of rows the
+import metadata file has and the amount of rows the `Import` at that time has
+processed. See the API documentation of this endpoint for more information.
+
+**Retrieving a report file**
+
+Whenever an `Import` instance has the status `finished` or `error` an report
+file can be downloaded of the process. This report is a csv almost identical to
+that of the provided metadata file with the additional columns which specify wether
+an row is imported or not and if there any comments about the row.
+
+Import behavior
+----------------
+
+The actual import process, which is a background task, imports each row in
+batches (configured through `IMPORT_DOCUMENTEN_BATCH_SIZE`). During each batch
+several error situations can occure. For example an existing
+`EnkelvoudigInformatieObject` can exist with the specified `uuid`. This can be
+considered a validation error and will not cause other rows in the specific batch
+not to be imported.
+
+Another situation that can occure is for example a database connection loss. This
+causes an situation where the import process cannot proceed and will stop the import
+process (the background task). In this situation the database cannot be reached and
+the data `Import` instance will be out-of-sync. Logging however is done and the
+report file will have comments for all rows in that specified batch.
+
+During the import, files are copied to the import path that the "regular" endpoint
+for `EnkelvoudigInformatieObject`'s would do so too. Files are however overwritten
+whenever they already exist.
```

Comment:
I would maybe rephrase this to something like:

```
If a row does not cause any validation errors, the file associated with that row will be copied to Open Zaak's storage. If the file already exists there, it will be overwritten.
```